.githooks/pre-push

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+npx lerna run lint
```

Comment:
This will just throw an error if any linter rule is not being followed. Neither will fix them nor will commit the changes.
We may want to:
- Run the linter
- `git add` the files modified by the linter and commit them with a generic commit message such as `Run lintfix`
- continue with the `git push`.

WDYT?

Reply:
@facuspagnuolo may I push some changes to your branch so we could close this PR? :hugs: